package mhud2;

option optimize_for = LITE_RUNTIME;

message Log
{
    enum LogType
    {
        LOG_INFO = 0;
        LOG_ERROR = 1;
    }

    optional LogType log_type = 1;
    optional string log_msg = 2;
}

message Preferences
{
    optional bool show_tears_fired = 1;
    optional bool show_shot_height = 2;
    optional int32 stat_precision = 3;
    optional bool split_deal_chance = 4;
}